lang/cs/build.props

```diff
@@ -38,7 +38,7 @@ under the License.
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>DEBUG;TRACE;WINDOWS</DefineConstants>
```

Comment:
@motus Answer to your question about testing functionality:

To test if this works, build with and without this WINDOWS constant. With WINDOWS defined, it should give you process-level information on a windows machine. Without WINDOWS defined, it should give you the "Unsupported" path.